pkg/module/all_test.go

```diff
@@ -13,6 +13,9 @@ const (
 	// github.com/NYTimes/gizmo is a example of a path that needs to be encoded so we can cover that case as well
 	repoURI = "github.com/NYTimes/gizmo"
 	version = "v0.1.4"
+
+	privateRepoURI     = "github.com/athens-artifacts/private"
```

Comment:
random, but why is this in a different org?

Reply:
@twexler https://github.com/athens-artifact is a dedicated organization for testing Athens where we put repositories with an exact state (i.e. number of semver versions, major SIV upgrades, private/public etc) and see how Athens would behave against them. This makes it so that testing against live repositories is predictable because if we tested against 3rd party libraries, they can end up being deleted or mutated in ways that break our tests. Most of the tests against this organization lives here: https://github.com/gomods/athens/blob/main/pkg/download/protocol_test.go
hope this helps :v:

Reply:
Gotcha, makes sense. Not _quite_ sure why it should be a different org, but the rest seems logical

Reply:
@twexler to keep `github.com/gomods` clean and only containing meaningful repositories for users (and not test artifacts)